ui-tests/tests/widget.test.ts

```diff
@@ -79,6 +79,99 @@ test('test create new connection', async ({ page }) => {
 
 });
 
+
+const labels_in_memory = ['DuckDB', 'SQLite']
+
+
+test('test irrelevant fields in memory db', async ({ page }) => {
+    await displayWidget(page);
+    await page.locator('#createNewConnection').click();
+    for (const label of labels_in_memory) {
```

Comment:
instead of using an internal for loop, you can parameterize (similar to pytest) so you create multiple tests for the given parameters. we already have some like that, you can check it

Reply:
example (I think you have access to this repo): https://github.com/ploomber/cloud-frontend/blob/b435f728c0a841579a34fca38ea9ff4749ec2da1/src/tests/playwright/register.spec.js#L64

Reply:
[fixed](https://github.com/ploomber/jupysql-plugin/pull/73/commits/769403be5bc29b39d4c6d84c8a81607db59b7110)